src/hud.rs

```diff
@@ -76,24 +76,6 @@ impl Hud {
                 .with_width(frame_size.0 as f32)
                 .with_height(frame_size.1 as f32)
                 .with_visibility(false)
-                .with_child(
```

Comment:
I'd put that in game options, something like "Show crosshair" instead of removing it completely.

Reply:
Ah okay. My thought was that the crosshair was redundant with the laser

Reply:
Actually laser was just a raycasting test :) I thought to remove it but then decided to leave it,